src/nodejs/src/audio-context-lazy.ts

```diff
@@ -0,0 +1,82 @@
+const LogScope = 'AudioContextLazy';
+
+export class AudioContextLazy {
+    private whenInitialized?: Promise<void> = null;
+    private audioContext?: AudioContext = null;
+
+    constructor() {
+        this.addInitEventListeners();
+    }
+
+    public async get(): Promise<AudioContext> {
+        if (!this.audioContext) {
+            await this.whenInitialized;
+        }
+
+        return this.audioContext;
```

Comment:
Probably we need to check state there and call resume if state is not expected

Reply:
done